Documentation/Storage-Configuration/Object-Storage-RGW/ceph-object-multisite.md

```diff
@@ -293,3 +293,84 @@ kubectl delete -f object-store.yaml
 ```
 
 Removing object store(s) from the master zone of the master zone group should be done with caution. When all of these object-stores are deleted the period cannot be updated and that realm cannot be pulled.
+
+#### Set up Multisite Configs for already existing CephObjectStore
+
+When a single site `CephObjectStore` CR request comes to Rook, it internally creates zone/zonegroup/realm with the same name as object store. So user can just export them into Kubernetes resources using the same name. For example consider following `CephObjectStore`
+```yaml
+apiVersion: ceph.rook.io/v1
```

Comment:
To keep it a bit simpler, I don't think we need this yaml snippet. We can just say we are starting with and existing object store.